.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: ci
+on: [push]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    name: Build and test
+    steps:
+      - uses: actions/checkout@v1
+      - uses: bahmutov/npm-install@v1
```

Comment:
Hmm, I always used regular `npm install` in Actions - any reason for an external action here?

Reply:
Ah, read the README, apparently it also caches stuff. I guess it's fine.